examples/tools/autotools/autotools_toolchain/build_project_autotools_toolchain.rst

```diff
@@ -0,0 +1,215 @@
+`Conan Center <https://conan.io/center/>`_ and install them as build requirements.
+
+Please, at first, clone the sources to recreate this project, you can find them in the
+`examples2.0 repository <https://github.com/conan-io/examples2>`_ in GitHub:
+
+.. code-block:: shell
+
+    git clone https://github.com/conan-io/examples2.git
+    cd examples2/examples/tools/autotools/autotoolstoolchain/string_formatter
+
+We start from a very simple C++ language project with this structure:
+
+.. code-block:: text
+
+    .
+    ├── configure.ac
+    ├── Makefile.am
+    ├── conanfile.txt
+    └── src
+        └── main.cpp
+
+This project contains a basic *configure.ac* including the **fmt** pkg-config dependency and the
+source code for the string formatter program in *main.cpp*.
+
+Let's have a look at the *main.cpp* file, it only prints a default message, but uses ``fmt::print`` method for it.
+
+.. code-block:: cpp
+    :caption: **main.c**
+
+    #include <cstdlib>
+    #include <fmt/core.h>
+
+    int main(int argc, char** argv) {
+        fmt::print("{} - The C++ Package Manager!\n", "Conan");
+        return EXIT_SUCCESS;
+    }
+
+The ``configure.ac`` tells us about our project will a C++ compiler and a pkg-config module named as ``fmt.pc``
+
+.. code-block:: text
+    :caption: **configure.ac**
+
+    AC_INIT([stringformatter], [0.1.0])
+    AM_INIT_AUTOMAKE([1.10 -Wall no-define foreign])
+    AC_CONFIG_SRCDIR([src/main.cpp])
+    AC_CONFIG_FILES([Makefile])
+    PKG_CHECK_MODULES([fmt], [fmt])
+    AC_PROG_CXX
+    AC_OUTPUT
+
+And, the *Makefile.am* has ``string_formatter`` as our expected executable, and it will linked to the ``fmt`` library.
+
+.. code-block:: text
+    :caption: **Mafile.am**
+
+    AUTOMAKE_OPTIONS = subdir-objects
+    ACLOCAL_AMFLAGS = ${ACLOCAL_FLAGS}
+
+    bin_PROGRAMS = string_formatter
+    string_formatter_SOURCES = src/main.cpp
+    string_formatter_CPPFLAGS = $(fmt_CFLAGS)
+    string_formatter_LDADD = $(fmt_LIBS)
+
+
+The *conanfile.txt* looks simple, just install **fmt** and uses two generatos to build our project
+
+.. code-block:: ini
+    :caption: **conanfile.txt**
+
+    [requires]
+    fmt/9.1.0
+
+    [generators]
+    AutotoolsToolchain
+    PkgConfigDeps
+
+In this case, we will use :ref:`PkgConfigDeps<conan_tools_gnu_pkgconfigdeps>` to generate information about where the **fmt** library
+files are installed thanks to the `*.pc` files and :ref:`AutotoolsToolchain<conan_tools_gnu_autotoolstoolchain>` to pass build information
+to *autotools* using a `conanbuild[.sh|.bat]` file that describes the compilation environment.
+
+We will use Conan to install **fmt** and generate the files that Autotools needs to find this library and build our project.
+
+
+Building on Linux and macOS
+---------------------------
+
+First, we should install the requirements. For this example we didn't consider your profile, but ``fmt`` is higly compatible with many different configurations.
+
```

Comment:
Maybe we can add more context to the sentence: "For this example we didn't consider your profile"

Reply:
reformulated, please, review it again.